diff --git a/ui/views/win/hwnd_message_handler.cc b/ui/views/win/hwnd_message_handler.cc
index e9197d95aa22f..2a20c0f78a7f6 100644
--- a/ui/views/win/hwnd_message_handler.cc
+++ b/ui/views/win/hwnd_message_handler.cc
@@ -994,7 +994,7 @@ void HWNDMessageHandler::FrameTypeChanged() {
   if (!custom_window_region_.is_valid() && IsFrameSystemDrawn()) {
     dwm_transition_desired_ = true;
   }
-  if (!dwm_transition_desired_ || !IsFullscreen()) {
+  if ((!dwm_transition_desired_ || !IsFullscreen()) && IsFrameSystemDrawn()) {
     PerformDwmTransition();
   }
 }
